tests/AutoReview/FixerFactoryTest.php

```diff
@@ -491,6 +491,7 @@ private static function getFixersPriorityGraph(): array
                 'not_operator_with_successor_space',
                 'php_unit_dedicate_assert',
                 'single_space_after_construct',
```

Comment:
Isn't it a part of `single_space_around_construct`?

Reply:
as long as both fixers exist, both should be tested. including the priority of them.

extra points for our self-tests to detect if it's missing and fail another test in such case.